skaffold.yaml

```diff
@@ -12,20 +12,38 @@ build:
     docker:
       dockerfile: Dockerfile
       buildArgs:
-        AEA_VERSION: "1.14.0"
+        AEA_VERSION: "1.14.0.post1"
         AEA_AGENT: "{{.AEA_AGENT}}"
 
 
 profiles:
-  - name: docs
+  - name: release
     build:
-      tagPolicy:
-        envTemplate:
-          template: "{{.VERSION}}"
-      artifacts:
-      - image: valory/open-autonomy-docs
-        docker:
-          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+        tagPolicy:
+          envTemplate:
+            template: "{{.VERSION}}"
+        artifacts:
+        - image: valory/open-autonomy-open-aea
+          context: deployments/Dockerfiles/open_aea/
+          docker:
+            noCache: true
+            dockerfile: Dockerfile
+            buildArgs:
+              AEA_VERSION: "1.14.0.post1"
```

Comment:
please double check this is the right hash

Reply:
You mean version?

Reply:
This comment was meant for the hello world agent